{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/utils/views.py

```
@@ -7,3 +18,36 @@ def get_template_names(self):
         template_name = self.kwargs.get('slug', '_base')
 
         return ['frontend/{}.html'.format(template_name)]
+
+
+class FrontendEditView(View):
+    def post(self, request):
+        if request.user.is_superuser:
+            data = json.loads(request.body.decode('utf-8'))
+            app_name = data.get('app', False)
+            model_name = data.get('model', False)
+            if app_name and model_name:
+                model = apps.get_model(app_label=app_name, model_name=model_name)
+                pk = data.get('pk', False)
+                field = data.get('field', False)
+                value = data.get('value', False)
+                if field and not model._meta.get_field(field).get_internal_type() == 'HtmlField':
+                    value = strip_tags(re.sub('<br>', '\n', value))
+                if field and model._meta.get_field(field).get_internal_type() == 'ImageRefField':
+                    value = File.objects.get(pk=value)
+                if model and pk and field:
+                    candidate = model.objects.filter(pk=pk).first()
+                    setattr(candidate, field, value)
+                    candidate.save()
+                    return JsonResponse({'success': True})
+        return JsonResponse({'success': False})
+
+    def get(self, request):
+        if request.user.is_superuser:
+            data = request.GET
+            image_pk = data.get('pk', False)
```

Don't handle a "can't happen" case - use `data['pk']` because this should throw an exception if it's not present.

---

(It also eliminates the branch below this line.)